java/src/hu/holyoil/neighbour/Asteroid.java

```diff
@@ -415,6 +415,15 @@ public TeleportGate GetTeleporter() {
 
     }
 
+    /**
+     * Felfedezetté teszi az aszteroidát.
+     * Átállítja az isDiscovered tagváltozót igazra.
+     */
+    public void Discover(){
```

Comment:
This is currently not used. I believe onAsteroid setter is the last function called after a successful move. This could be used in Settlers to signal a discovery.